### PR TITLE
Format Go code

### PR DESCRIPTION
**Pull request by mlafeldt**
By using `gofmt -w -s`.

### CODE REVIEW

File: syslog/syslog_test.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func handle(conn io.ReadCloser, messages chan string) {
 
 func generatePackets() []Packet {
 	packets := make([]Packet, 10)
-	for i, _ := range packets {
```
Comment (troy) [inline syslog/syslog_test.go]: Is the blank identifier automatic here?

**Comment by troy** [comment]
@mlafeldt (just in case you don't see it): I added 1 comment inline for you.

**Comment by mlafeldt** [comment]
Yes, it's automatic. `gofmt -s` will simplify code and this is one of the simplifications it applied.

**Comment by troy** [comment]
Works for me. @leonsodhi all yours.

**Comment by mlafeldt** [comment]
Rebased on top of master. If you want to merge it, now would be the perfect time. :smile:

**Comment by eric** [comment]
:+1:

**Comment by leonsodhi** [comment]
thanks, @mlafeldt! Mergin'

**Pull request merged**